_includes/outreach-partners.html

```diff
@@ -0,0 +1,11 @@
+<div id="outreach-partners" class="page-wrapper sponsor-section">
+    <div class="page-details">
+        <div class="title">Outreach Partners</div>
+        <section class="outreach-grayscale">
+            <div class="row">
+                <div class="col s4"><a href="https://fossunited.org" target="_blank"><img class="responsive-img"
```

Comment:
I would prefer an indentation like:

```HTML
<div>
  <a>
    <img />
  </a>
</div>
```

Also, please add `rel="nofollow noreferrer"` to avoid target blank vulnerability.

Reply:
Done